pkg/util/cluster.go

```diff
@@ -142,6 +148,24 @@ func (c *ClusterInfoImp) initOpenshift(ctx context.Context, cl client.Client) er
 
 	c.controlPlaneHighlyAvailable = clusterInfrastructure.Status.ControlPlaneTopology == openshiftconfigv1.HighlyAvailableTopologyMode
 	c.infrastructureHighlyAvailable = clusterInfrastructure.Status.InfrastructureTopology == openshiftconfigv1.HighlyAvailableTopologyMode
+
+	clusterNetwork := &openshiftconfigv1.Network{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+	}
+	err = cl.Get(ctx, client.ObjectKeyFromObject(clusterNetwork), clusterNetwork)
+	if err != nil {
+		return err
+	}
+	cn := clusterNetwork.Status.ClusterNetwork
+	for _, i := range cn {
+		c.logger.Info("Cluster Network",
+			"CIDR", i.CIDR,
+			"Host Prefix", i.HostPrefix,
+		)
+	}
+	c.singlestackipv6 = len(cn) == 1 && strings.Contains(cn[0].CIDR, ":")
```

Comment:
I'm not 100% sure if this is the best way to detect if the OCP cluster is configured as single stack IPv6, but it's my best guess after reading https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html.

Open to other/better/canonical approaches!